apps/web/composables/useHomepage/types.ts

```diff
@@ -66,7 +66,7 @@ export interface UseHomepageDataState {
 }
 
 export interface UseHomepage {
-  data: Readonly<Ref<UseHomepageDataState['data']>>;
+  data: Ref<UseHomepageDataState['data']>;
```

Comment:
**question**: Why is this no longer `Readonly`?

Reply:
Because when we are adding a new block, we are basically changing the state of it, so we write in there.
If it was only Readonly, we could not add new blocks.